### ISSUE TITLE
qwen support long chat

### ISSUE THREAD

**Issue by allenhaozi** [vllm-project/vllm#1323]
support long chat [Support Longchat #555](https://github.com/vllm-project/vllm/pull/555) has been merged
This does not support qwen yet. 

If I refer to qwen's [ntk_alpha method](https://huggingface.co/Qwen/Qwen-7B-Chat/blob/main/modeling_qwen.py#L779), can vllm support qwen's long chat? thank you

**Comment by allenhaozi** [comment]
https://github.com/QwenLM/Qwen/issues/398

**Comment by Sanster** [comment]
rope_scaling support for qwen has already been merged into the main branch: https://github.com/vllm-project/vllm/pull/1210

```json
"rope_scaling": {
    "type": "dynamic",
    "factor": 2.0
  }
```

**Comment by allenhaozi** [comment]
> rope_scaling support for qwen has already been merged into the main branch: #1210
> 
> ```json
> "rope_scaling": {
>     "type": "dynamic",
>     "factor": 2.0
>   }
> ```

@Sanster Thank you for your reply

There are some differences in the implementation of [qwen ntk](https://huggingface.co/Qwen/Qwen-7B-Chat/blob/main/modeling_qwen.py#L779), 
I would like to discuss it
